README.md

```diff
@@ -27,21 +25,38 @@ def deps do
 end
 ```
 
-## Usage
+## Configuration
 
-Add the Firebase auth issuer name for your project to your `config.exs`. This is required to make sure only your project's firebase tokens are accepted.
+Configure the ID Token issuer in your `config.exs`. This is required to ensure that only ID Tokens from your own project will be accepted:
 
 ```elixir
 config :ex_firebase_auth, :issuer, "https://securetoken.google.com/project-123abc"
 ```
 
-Verifying a token
+## Usage
+
+To verify an ID Token, use `ExFirebaseAuth.Token.verify_token/1`:
 
 ```elixir
-ExFirebaseAuth.Token.verify_token("Some token string")
-iex> {:ok, "userid", %{}}
+
+iex(1)> ExFirebaseAuth.Token.verify_token("<Some valid Firebase ID Token goes here>")
+{:ok, "<User UID goes here>",
+  %JOSE.JWT{
+    fields: %{
+      "email" => "jose@valim.com",
+      "email_verified" => true,
+      "name" => "<User name>",
+      "picture" => "<User picture URL>",
+      "user_id" => "<User UID>",
+      # (...) among other fields (...)
+    }
+  }}
```

Comment:
This portion might help people looking for ways to extract useful information from ID Tokens.  I recently found myself inspecting it manually to understand if/how I could extract information from the JWT instead of somehow else, so I think the `README.md` could be made home for this information.